jsonld-schema/sb/context.json

```diff
@@ -1,7 +1,7 @@
 {
   "@context": {
     "scheme": "http://schema.org",
-    "sb": "https://raw.githubusercontent.com/project-sunbird/sunbird-dial-service/release-5.0.0/jsonld-schema/sb/schema.jsonld#",
+    "sb": "https://sunbirdstagingpublic.blob.core.windows.net/sunbird-dial-staging/jsonld-schema/local/sb/schema.jsonld",
```

Comment:
This should be variabalized later. So that specific to the environment, it will take the blob file.

Reply:
Not sure if we can implement that inside context.json file. @AmiableAnil ??

Reply:
@vinukumar-vs @Jayaprakash8887 we can variabalized it as a part of jenkins job while uploading to blob. Thats possible.

Reply:
@vinukumar-vs But SB vocabulary is single source of truth. It shouldnt be varying based on environments is what I feel.